homework/04LessonFour.js

```diff
@@ -87,7 +101,7 @@ var nestedObjects = {
 // What is an associative array? What is the difference between an array and an associative array?
 //
 // Your Answer Goes Here:
-//
+//An associate array (also known as Objects) like an array is a colletion of data. They differ in that the associated array utilizes keys (anexplicit index) with associated values and the array has an explicit index that start with 0.
 //
```

Comment:
Thank you! I made the correction and will commit this evening.

Sent from my iPhone

> On Aug 21, 2016, at 8:54 PM, maddiesolz <notifications@github.com> wrote:
> 
> In homework/04LessonFour.js:
> 
> > @@ -87,7 +101,7 @@ var nestedObjects = {
> >  // What is an associative array? What is the difference between an array and an associative array?
> >  //
> >  // Your Answer Goes Here:
> > -//
> > +//An associate array (also known as Objects) like an array is a colletion of data. They differ in that the associated array utilizes keys (anexplicit index) with associated values and the array has an explicit index that start with 0.
> >  //
> Great work!! Fix the ';' and you got it
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>